[fltk] Undefined symbols on macOS 11.1 (20C69)

**Is your feature request related to a problem? Please describe.**

### `CMakeLists.txt`
```cmake
        cmake_minimum_required(VERSION 3.0)
cmake_policy(SET CMP0048 NEW)
project(pp VERSION 0.0.0 LANGUAGES CXX)

set(src src/main.cpp)

find_package(FLTK CONFIG REQUIRED)
# mark_as_advanced (FLTK)

add_executable(${PROJECT_NAME} ${src})
target_link_libraries(${PROJECT_NAME} PRIVATE fltk fltk_gl fltk_forms fltk_images)
````

### `src/main.cpp`
https://www.fltk.org/doc-1.3/basics.html
```c++
#include <FL/Fl.H>
#include <FL/Fl_Window.H>
#include <FL/Fl_Box.H>
int main(int argc, char **argv) {
    Fl_Window *window = new Fl_Window(340,180);
    Fl_Box *box = new Fl_Box(20,40,300,100,"Hello, World!");
    box->box(FL_UP_BOX);
    box->labelfont(FL_BOLD+FL_ITALIC);
    box->labelsize(36);
    box->labeltype(FL_SHADOW_LABEL);
    window->end();
    window->show(argc, argv);
    return Fl::run();
}
```

### `shell`
```sh
        $ [path]/vcpkg install fltk
        $ mkdir build && cd $_
$ cmake .. -DCMAKE_TOOLCHAIN_FILE=[path]/vcpkg/scripts/buildsystems/vcpkg.cmake
        $ cmake --build .
====================[ Build | pp | Debug ]===========================
"CLion/ch-0/203.7717.62/CLion.app/Contents/bin/cmake/mac/bin/cmake" --build pp/cmake-build-debug --target pp -- -j 6
"CLion/ch-0/203.7717.62/CLion.app/Contents/bin/cmake/mac/bin/cmake" -Spp -Bpp/cmake-build-debug --check-build-system CMakeFiles/Makefile.cmake 0
/Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/Makefile2 pp
"CLion/ch-0/203.7717.62/CLion.app/Contents/bin/cmake/mac/bin/cmake" -Spp -Bpp/cmake-build-debug --check-build-system CMakeFiles/Makefile.cmake 0
"CLion/ch-0/203.7717.62/CLion.app/Contents/bin/cmake/mac/bin/cmake" -E cmake_progress_start pp/cmake-build-debug/CMakeFiles 2
/Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/Makefile2 CMakeFiles/pp.dir/all
/Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/pp.dir/build.make CMakeFiles/pp.dir/depend
        cd pp/cmake-build-debug && "CLion/ch-0/203.7717.62/CLion.app/Contents/bin/cmake/mac/bin/cmake" -E cmake_depends "Unix Makefiles" pp pp pp/cmake-build-debug pp/cmake-build-debug pp/cmake-build-debug/CMakeFiles/pp.dir/DependInfo.cmake --color=
Scanning dependencies of target pp
/Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/pp.dir/build.make CMakeFiles/pp.dir/build
[ 50%] Building CXX object CMakeFiles/pp.dir/src/main.cpp.o
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++   -isystem vcpkg/installed/x64-osx/include  -g -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk   -std=gnu++17 -o CMakeFiles/pp.dir/src/main.cpp.o -c pp/src/main.cpp
[100%] Linking CXX executable pp
"CLion/ch-0/203.7717.62/CLion.app/Contents/bin/cmake/mac/bin/cmake" -E cmake_link_script CMakeFiles/pp.dir/link.txt --verbose=1
                                                                                                                                                                                               /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -g -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/pp.dir/src/main.cpp.o  -o pp  vcpkg/installed/x64-osx/debug/lib/libfltk.a vcpkg/installed/x64-osx/debug/lib/libfltk_gl.a vcpkg/installed/x64-osx/debug/lib/libfltk_forms.a vcpkg/installed/x64-osx/debug/lib/libfltk_images.a -framework OpenGL vcpkg/installed/x64-osx/debug/lib/libfltk.a /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib/libdl.tbd vcpkg/installed/x64-osx/debug/lib/libjpeg.a vcpkg/installed/x64-osx/debug/lib/libz.a vcpkg/installed/x64-osx/debug/lib/libpng16d.a vcpkg/installed/x64-osx/debug/lib/libz.a vcpkg/installed/x64-osx/debug/lib/libpng16d.a
        Undefined symbols for architecture x86_64:
"_CFAbsoluteTimeGetCurrent", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
Fl::repeat_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
"_CFAttributedStringCreate", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
variation_selector_width(__CFString const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CFDataCreate", referenced from:
Fl::copy(char const*, int, int, char const*) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::dnd(int) in libfltk.a(Fl_cocoa.mm.o)
"_CFDictionaryCreateMutable", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_CFDictionarySetValue", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
variation_selector_width(__CFString const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CFGetRetainCount", referenced from:
bmProviderRelease(void*, void const*, unsigned long) in libfltk.a(Fl_Double_Window.cxx.o)
fl_delete_offscreen(CGContext*) in libfltk.a(Fl_Double_Window.cxx.o)
"_CFNumberCreate", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_CFRelease", referenced from:
delete_timer(MacTimeout&) in libfltk.a(Fl_cocoa.mm.o)
-[FLAppDelegate applicationDidUpdate:] in libfltk.a(Fl_cocoa.mm.o)
-[FLView performDragOperation:] in libfltk.a(Fl_cocoa.mm.o)
q_set_window_title(NSWindow*, char const*, char const*) in libfltk.a(Fl_cocoa.mm.o)
Fl_Copy_Surface::complete_copy_pdf_and_tiff() in libfltk.a(Fl_cocoa.mm.o)
Fl::copy(char const*, int, int, char const*) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
...
"_CFRetain", referenced from:
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
"_CFRunLoopAddTimer", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
"_CFRunLoopGetCurrent", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
delete_timer(MacTimeout&) in libfltk.a(Fl_cocoa.mm.o)
"_CFRunLoopRemoveTimer", referenced from:
delete_timer(MacTimeout&) in libfltk.a(Fl_cocoa.mm.o)
"_CFRunLoopTimerCreate", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
"_CFRunLoopTimerGetNextFireDate", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
"_CFRunLoopTimerSetNextFireDate", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
Fl::repeat_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
"_CFStringCreateByCombiningStrings", referenced from:
-[FLView performDragOperation:] in libfltk.a(Fl_cocoa.mm.o)
"_CFStringCreateMutableWithExternalCharactersNoCopy", referenced from:
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
"_CFStringCreateWithBytes", referenced from:
get_plain_text_from_clipboard(int) in libfltk.a(Fl_cocoa.mm.o)
"_CFStringCreateWithCString", referenced from:
q_set_window_title(NSWindow*, char const*, char const*) in libfltk.a(Fl_cocoa.mm.o)
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_CFStringCreateWithCharacters", referenced from:
q_set_window_title(NSWindow*, char const*, char const*) in libfltk.a(Fl_cocoa.mm.o)
fl_mac_width(unsigned short const*, int, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CFStringCreateWithCharactersNoCopy", referenced from:
fl_mac_width(unsigned short const*, int, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
surrogate_width(unsigned short const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CFStringGetCString", referenced from:
-[FLView performDragOperation:] in libfltk.a(Fl_cocoa.mm.o)
"_CFStringGetCharacters", referenced from:
cocoaKeyboardHandler(NSEvent*) in libfltk.a(Fl_cocoa.mm.o)
"_CFStringGetLength", referenced from:
-[FLView performDragOperation:] in libfltk.a(Fl_cocoa.mm.o)
"_CFStringGetMaximumSizeForEncoding", referenced from:
-[FLView performDragOperation:] in libfltk.a(Fl_cocoa.mm.o)
"_CFUUIDCreate", referenced from:
Fl_Preferences::newUUID() in libfltk.a(Fl_Preferences.cxx.o)
"_CFUUIDGetUUIDBytes", referenced from:
Fl_Preferences::newUUID() in libfltk.a(Fl_Preferences.cxx.o)
"_CGBitmapContextCreate", referenced from:
prepare_bitmap_for_layer(int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::create_offscreen_with_alpha(int, int) in libfltk.a(Fl_Double_Window.cxx.o)
fl_create_offscreen(int, int) in libfltk.a(Fl_Double_Window.cxx.o)
"_CGBitmapContextCreateImage", referenced from:
-[FLViewLayer displayLayer:] in libfltk.a(Fl_cocoa.mm.o)
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep_layer(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGBitmapContextGetAlphaInfo", referenced from:
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
"_CGBitmapContextGetBitsPerPixel", referenced from:
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
fl_read_image(unsigned char*, int, int, int, int, int) in libfltk.a(fl_read_image.cxx.o)
"_CGBitmapContextGetBytesPerRow", referenced from:
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
fl_read_image(unsigned char*, int, int, int, int, int) in libfltk.a(fl_read_image.cxx.o)
"_CGBitmapContextGetData", referenced from:
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
fl_delete_offscreen(CGContext*) in libfltk.a(Fl_Double_Window.cxx.o)
fl_read_image(unsigned char*, int, int, int, int, int) in libfltk.a(fl_read_image.cxx.o)
"_CGBitmapContextGetHeight", referenced from:
Fl_X::q_fill_context() in libfltk.a(Fl_cocoa.mm.o)
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
fl_read_image(unsigned char*, int, int, int, int, int) in libfltk.a(fl_read_image.cxx.o)
"_CGBitmapContextGetWidth", referenced from:
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
fl_read_image(unsigned char*, int, int, int, int, int) in libfltk.a(fl_read_image.cxx.o)
"_CGColorCreate", referenced from:
flcolortocgcolor(unsigned int) in libfltk.a(fl_font.cxx.o)
"_CGColorSpaceCreateDeviceGray", referenced from:
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
innards(unsigned char const*, int, int, int, int, int, int, int, void (*)(void*, int, int, int, unsigned char*), void*) in libfltk.a(fl_draw_image.cxx.o)
"_CGColorSpaceCreateDeviceRGB", referenced from:
prepare_bitmap_for_layer(int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::CGImage_from_window_rect(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::create_offscreen_with_alpha(int, int) in libfltk.a(Fl_Double_Window.cxx.o)
fl_create_offscreen(int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
...
"_CGColorSpaceCreateWithName", referenced from:
flcolortocgcolor(unsigned int) in libfltk.a(fl_font.cxx.o)
"_CGColorSpaceRelease", referenced from:
prepare_bitmap_for_layer(int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::CGImage_from_window_rect(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::create_offscreen_with_alpha(int, int) in libfltk.a(Fl_Double_Window.cxx.o)
fl_create_offscreen(int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
...
"_CGContextAddArc", referenced from:
Fl_Graphics_Driver::circle(double, double, double) in libfltk.a(fl_vertex.cxx.o)
Fl_Graphics_Driver::arc(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
Fl_Graphics_Driver::pie(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
"_CGContextAddArcToPoint", referenced from:
Fl_X::clip_to_rounded_corners(CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGContextAddLineToPoint", referenced from:
Fl_X::clip_to_rounded_corners(CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::xyline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
...
"_CGContextClearRect", referenced from:
prepare_bitmap_for_layer(int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGContextClip", referenced from:
Fl_X::clip_to_rounded_corners(CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGContextClipToMask", referenced from:
Fl_Window::draw() in libfltk.a(Fl_Window_shape.cxx.o)
"_CGContextClipToRect", referenced from:
Fl_Window::make_current() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_begin_image(CGRect&, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::draw_scaled(Fl_Image*, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
"_CGContextClipToRects", referenced from:
Fl_Graphics_Driver::restore_clip() in libfltk.a(fl_rect.cxx.o)
"_CGContextClosePath", referenced from:
Fl_Graphics_Driver::loop(int, int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::loop(int, int, int, int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::polygon(int, int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::polygon(int, int, int, int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::end_polygon() in libfltk.a(fl_vertex.cxx.o)
Fl_Graphics_Driver::end_complex_polygon() in libfltk.a(fl_vertex.cxx.o)
Fl_Graphics_Driver::pie(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
...
"_CGContextDrawImage", referenced from:
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
innards(unsigned char const*, int, int, int, int, int, int, int, void (*)(void*, int, int, int, unsigned char*), void*) in libfltk.a(fl_draw_image.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_Bitmap*, int, int, int, int, int, int) in libfltk.a(Fl_Bitmap.cxx.o)
Fl_System_Printer::print_window_part(Fl_Window*, int, int, int, int, int, int) in libfltk.a(Fl_Quartz_Printer.mm.o)
"_CGContextEndPage", referenced from:
Fl_Copy_Surface::complete_copy_pdf_and_tiff() in libfltk.a(Fl_cocoa.mm.o)
"_CGContextFillPath", referenced from:
Fl_Graphics_Driver::polygon(int, int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::polygon(int, int, int, int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::end_polygon() in libfltk.a(fl_vertex.cxx.o)
Fl_Graphics_Driver::end_complex_polygon() in libfltk.a(fl_vertex.cxx.o)
Fl_Graphics_Driver::circle(double, double, double) in libfltk.a(fl_vertex.cxx.o)
Fl_Graphics_Driver::pie(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
"_CGContextFillRect", referenced from:
Fl_X::draw_layer_to_context(void*, CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::rectf(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
...
"_CGContextFlush", referenced from:
Fl::flush() in libfltk.a(Fl.cxx.o)
Fl_X::q_release_context(Fl_X*) in libfltk.a(Fl_cocoa.mm.o)
fl_end_offscreen() in libfltk.a(Fl_Double_Window.cxx.o)
Fl_System_Printer::end_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_CGContextGetCTM", referenced from:
Fl_X::q_fill_context() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_begin_image(CGRect&, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGContextMoveToPoint", referenced from:
Fl_X::clip_to_rounded_corners(CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::xyline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
...
"_CGContextRelease", referenced from:
-[FLViewLayer viewFrameDidChange] in libfltk.a(Fl_cocoa.mm.o)
-[FLViewLayer dealloc] in libfltk.a(Fl_cocoa.mm.o)
Fl_Copy_Surface::complete_copy_pdf_and_tiff() in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
fl_delete_offscreen(CGContext*) in libfltk.a(Fl_Double_Window.cxx.o)
"_CGContextRestoreGState", referenced from:
Fl_X::q_release_context(Fl_X*) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_clear_clipping() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_end_image() in libfltk.a(Fl_cocoa.mm.o)
Fl_Copy_Surface::complete_copy_pdf_and_tiff() in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::draw_layer_to_context(void*, CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Window::draw() in libfltk.a(Fl_Window_shape.cxx.o)
...
"_CGContextRotateCTM", referenced from:
Fl_Quartz_Graphics_Driver::draw(int, char const*, int, int, int) in libfltk.a(fl_font.cxx.o)
Fl_System_Printer::origin(int, int) in libfltk.a(Fl_Quartz_Printer.mm.o)
Fl_System_Printer::scale(float, float) in libfltk.a(Fl_Quartz_Printer.mm.o)
Fl_System_Printer::rotate(float) in libfltk.a(Fl_Quartz_Printer.mm.o)
"_CGContextSaveGState", referenced from:
Fl_Window::make_current() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_clear_clipping() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_begin_image(CGRect&, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::draw_layer_to_context(void*, CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Window::draw() in libfltk.a(Fl_Window_shape.cxx.o)
fl_begin_offscreen(CGContext*) in libfltk.a(Fl_Double_Window.cxx.o)
...
"_CGContextScaleCTM", referenced from:
Fl_Window::make_current() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_fill_context() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_begin_image(CGRect&, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::draw_scaled(Fl_Image*, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
Fl_Graphics_Driver::arc(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
Fl_Graphics_Driver::pie(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
...
"_CGContextSetLineCap", referenced from:
fl_quartz_restore_line_style_() in libfltk.a(fl_line_style.cxx.o)
"_CGContextSetLineDash", referenced from:
fl_quartz_restore_line_style_() in libfltk.a(fl_line_style.cxx.o)
"_CGContextSetLineJoin", referenced from:
fl_quartz_restore_line_style_() in libfltk.a(fl_line_style.cxx.o)
"_CGContextSetLineWidth", referenced from:
fl_quartz_restore_line_style_() in libfltk.a(fl_line_style.cxx.o)
"_CGContextSetRGBFillColor", referenced from:
Fl_Quartz_Graphics_Driver::color(unsigned int) in libfltk.a(fl_color.cxx.o)
Fl_Quartz_Graphics_Driver::color(unsigned char, unsigned char, unsigned char) in libfltk.a(fl_color.cxx.o)
imageFromText(char const*, int*, int*) in libfltk.a(Fl_cocoa.mm.o)
defaultDragImage(int*, int*) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::draw_layer_to_context(void*, CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGContextSetRGBStrokeColor", referenced from:
Fl_Quartz_Graphics_Driver::color(unsigned int) in libfltk.a(fl_color.cxx.o)
Fl_Quartz_Graphics_Driver::color(unsigned char, unsigned char, unsigned char) in libfltk.a(fl_color.cxx.o)
defaultDragImage(int*, int*) in libfltk.a(Fl_cocoa.mm.o)
"_CGContextSetShouldAntialias", referenced from:
Fl_Window::make_current() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::draw_layer_to_context(void*, CGContext*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::rect(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int) in libfltk.a(fl_rect.cxx.o)
...
"_CGContextSetTextMatrix", referenced from:
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
"_CGContextSetTextPosition", referenced from:
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
"_CGContextStrokePath", referenced from:
Fl_Graphics_Driver::xyline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::xyline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::yxline(int, int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::line(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
...
"_CGContextStrokeRect", referenced from:
Fl_Graphics_Driver::rect(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
"_CGContextTranslateCTM", referenced from:
Fl_Window::make_current() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_fill_context() in libfltk.a(Fl_cocoa.mm.o)
Fl_X::q_begin_image(CGRect&, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::draw(int, char const*, int, int, int) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::draw_scaled(Fl_Image*, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
Fl_Graphics_Driver::arc(int, int, int, int, double, double) in libfltk.a(fl_arci.cxx.o)
...
"_CGDataProviderCreateWithData", referenced from:
Fl_X::CGImage_from_window_rect(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Window::shape_bitmap_(Fl_Image*) in libfltk.a(Fl_Window_shape.cxx.o)
Fl_Window::shape_alpha_(Fl_Image*, int) in libfltk.a(Fl_Window_shape.cxx.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
innards(unsigned char const*, int, int, int, int, int, int, int, void (*)(void*, int, int, int, unsigned char*), void*) in libfltk.a(fl_draw_image.cxx.o)
fl_create_bitmask(int, int, unsigned char const*) in libfltk.a(Fl_Bitmap.cxx.o)
...
"_CGDataProviderRelease", referenced from:
Fl_X::CGImage_from_window_rect(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
innards(unsigned char const*, int, int, int, int, int, int, int, void (*)(void*, int, int, int, unsigned char*), void*) in libfltk.a(fl_draw_image.cxx.o)
fl_create_bitmask(int, int, unsigned char const*) in libfltk.a(Fl_Bitmap.cxx.o)
"_CGDisplayBounds", referenced from:
screen_init() in libfltk.a(screen_xywh.cxx.o)
"_CGDisplayScreenSize", referenced from:
screen_init() in libfltk.a(screen_xywh.cxx.o)
"_CGEventSourceKeyState", referenced from:
Fl::get_key(int) in libfltk.a(Fl_get_key.cxx.o)
"_CGGetActiveDisplayList", referenced from:
screen_init() in libfltk.a(screen_xywh.cxx.o)
"_CGImageCreate", referenced from:
Fl_X::CGImage_from_window_rect(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
innards(unsigned char const*, int, int, int, int, int, int, int, void (*)(void*, int, int, int, unsigned char*), void*) in libfltk.a(fl_draw_image.cxx.o)
"_CGImageCreateWithImageInRect", referenced from:
rect_to_NSBitmapImageRep_layer(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGImageGetShouldInterpolate", referenced from:
Fl_Quartz_Graphics_Driver::draw(Fl_RGB_Image*, int, int, int, int, int, int) in libfltk.a(Fl_Image.cxx.o)
"_CGImageMaskCreate", referenced from:
Fl_Window::shape_bitmap_(Fl_Image*) in libfltk.a(Fl_Window_shape.cxx.o)
Fl_Window::shape_alpha_(Fl_Image*, int) in libfltk.a(Fl_Window_shape.cxx.o)
fl_create_bitmask(int, int, unsigned char const*) in libfltk.a(Fl_Bitmap.cxx.o)
"_CGImageRelease", referenced from:
Fl_Window::~Fl_Window() in libfltk.a(Fl_Window.cxx.o)
-[FLViewLayer displayLayer:] in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep_layer(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Quartz_Graphics_Driver::copy_offscreen(int, int, int, int, CGContext*, int, int) in libfltk.a(Fl_Double_Window.cxx.o)
Fl_RGB_Image::~Fl_RGB_Image() in libfltk.a(Fl_Image.cxx.o)
Fl_RGB_Image::uncache() in libfltk.a(Fl_Image.cxx.o)
innards(unsigned char const*, int, int, int, int, int, int, int, void (*)(void*, int, int, int, unsigned char*), void*) in libfltk.a(fl_draw_image.cxx.o)
...
"_CGImageRetain", referenced from:
Fl_X::CGImage_from_window_rect(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGRectContainsRect", referenced from:
Fl_Widget::damage(unsigned char, int, int, int, int) in libfltk.a(Fl.cxx.o)
"_CGRectEqualToRect", referenced from:
-[FLWindow checkSubwindowFrame] in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::clip_box(int, int, int, int, int&, int&, int&, int&) in libfltk.a(fl_rect.cxx.o)
"_CGRectIntersection", referenced from:
-[FLWindow checkSubwindowFrame] in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::intersect_region_and_rect(flCocoaRegion*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::not_clipped(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::clip_box(int, int, int, int, int&, int&, int&, int&) in libfltk.a(fl_rect.cxx.o)
"_CGRectIsEmpty", referenced from:
Fl_X::intersect_region_and_rect(flCocoaRegion*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Graphics_Driver::not_clipped(int, int, int, int) in libfltk.a(fl_rect.cxx.o)
Fl_Graphics_Driver::clip_box(int, int, int, int, int&, int&, int&, int&) in libfltk.a(fl_rect.cxx.o)
"_CGRectIsNull", referenced from:
rect_to_NSBitmapImageRep(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_CGRectOffset", referenced from:
Fl_Window::make_current() in libfltk.a(Fl_cocoa.mm.o)
"_CGRectUnion", referenced from:
Fl_Graphics_Driver::clip_box(int, int, int, int, int&, int&, int&, int&) in libfltk.a(fl_rect.cxx.o)
"_CGShieldingWindowLevel", referenced from:
modal_window_level() in libfltk.a(Fl_cocoa.mm.o)
non_modal_window_level() in libfltk.a(Fl_cocoa.mm.o)
"_CTFontCreateForString", referenced from:
fl_mac_width(unsigned short const*, int, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
surrogate_width(unsigned short const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CTFontCreateWithName", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_CTFontGetAdvancesForGlyphs", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
fl_mac_width(unsigned short const*, int, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
surrogate_width(unsigned short const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CTFontGetAscent", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_CTFontGetDescent", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_CTFontGetGlyphsForCharacters", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
fl_mac_width(unsigned short const*, int, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
surrogate_width(unsigned short const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CTLineCreateWithAttributedString", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
variation_selector_width(__CFString const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CTLineDraw", referenced from:
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
"_CTLineGetImageBounds", referenced from:
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
"_CTLineGetOffsetForStringIndex", referenced from:
variation_selector_width(__CFString const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_CTLineGetTypographicBounds", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_NSApp", referenced from:
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
DataReady::DataReadyThread(void*) in libfltk.a(Fl_cocoa.mm.o)
fl_ready() in libfltk.a(Fl_cocoa.mm.o)
-[FLWindow waitForExpose] in libfltk.a(Fl_cocoa.mm.o)
do_queued_events(double) in libfltk.a(Fl_cocoa.mm.o)
-[FLTextView insertText:] in libfltk.a(Fl_cocoa.mm.o)
-[FLTextView doCommandBySelector:] in libfltk.a(Fl_cocoa.mm.o)
...
(maybe you meant: __OBJC_LABEL_PROTOCOL_$_NSApplicationDelegate, __OBJC_PROTOCOL_$_NSApplicationDelegate )
"_NSBeep", referenced from:
fl_beep(int) in libfltk.a(fl_ask.cxx.o)
"_NSDefaultRunLoopMode", referenced from:
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
fl_ready() in libfltk.a(Fl_cocoa.mm.o)
do_queued_events(double) in libfltk.a(Fl_cocoa.mm.o)
Fl_Window::wait_for_expose() in libfltk.a(Fl_cocoa.mm.o)
"_NSDeviceRGBColorSpace", referenced from:
Fl_X::set_cursor(Fl_RGB_Image const*, int, int) in libfltk.a(Fl_cocoa.mm.o)
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
GL_rect_to_nsbitmap(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
scale_nsbitmapimagerep(NSBitmapImageRep*, float) in libfltk.a(Fl_cocoa.mm.o)
"_NSDeviceWhiteColorSpace", referenced from:
Fl_X::set_cursor(Fl_RGB_Image const*, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_NSDisableScreenUpdates", referenced from:
fl_mac_flush_and_wait(double) in libfltk.a(Fl_cocoa.mm.o)
"_NSDragPboard", referenced from:
Fl_X::dnd(int) in libfltk.a(Fl_cocoa.mm.o)
"_NSEnableScreenUpdates", referenced from:
fl_mac_flush_and_wait(double) in libfltk.a(Fl_cocoa.mm.o)
"_NSEqualRects", referenced from:
-[FLWindow setSubwindowFrame] in libfltk.a(Fl_cocoa.mm.o)
"_NSFilenamesPboardType", referenced from:
-[FLView performDragOperation:] in libfltk.a(Fl_cocoa.mm.o)
Fl_X::make(Fl_Window*) in libfltk.a(Fl_cocoa.mm.o)
"_NSFontAttributeName", referenced from:
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
"_NSGraphicsContextDestinationAttributeName", referenced from:
scale_nsbitmapimagerep(NSBitmapImageRep*, float) in libfltk.a(Fl_cocoa.mm.o)
"_NSPDFPboardType", referenced from:
___cxx_global_var_init.824 in libfltk.a(Fl_cocoa.mm.o)
"_NSPICTPboardType", referenced from:
___cxx_global_var_init.834 in libfltk.a(Fl_cocoa.mm.o)
"_NSPasteboardTypePDF", referenced from:
___cxx_global_var_init.824 in libfltk.a(Fl_cocoa.mm.o)
"_NSPasteboardTypeString", referenced from:
___cxx_global_var_init.821 in libfltk.a(Fl_cocoa.mm.o)
"_NSPasteboardTypeTIFF", referenced from:
___cxx_global_var_init.825 in libfltk.a(Fl_cocoa.mm.o)
"_NSRectFill", referenced from:
scale_nsbitmapimagerep(NSBitmapImageRep*, float) in libfltk.a(Fl_cocoa.mm.o)
"_NSStringPboardType", referenced from:
___cxx_global_var_init.821 in libfltk.a(Fl_cocoa.mm.o)
"_NSTIFFPboardType", referenced from:
___cxx_global_var_init.825 in libfltk.a(Fl_cocoa.mm.o)
"_NSWindowDidMoveNotification", referenced from:
-[FLAppDelegate applicationDidChangeScreenParameters:] in libfltk.a(Fl_cocoa.mm.o)
"_NSWindowWillCloseNotification", referenced from:
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
"_NSZeroPoint", referenced from:
Fl_X::make(Fl_Window*) in libfltk.a(Fl_cocoa.mm.o)
"_NSZeroSize", referenced from:
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSApplication", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSArray", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSAttributedString", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSAutoreleasePool", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSBitmapImageRep", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSBundle", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSColor", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSCursor", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSDate", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSDictionary", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSEvent", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSFont", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSGraphicsContext", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSImage", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSMenu", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSMenuItem", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSMutableArray", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSNotificationCenter", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSObject", referenced from:
_OBJC_CLASS_$_FLWindowDelegate in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLAppDelegate in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLApplication in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLTextInputContext in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLaboutItemTarget in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSOpenGLContext", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSOpenGLPixelFormat", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSPasteboard", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSPrintInfo", referenced from:
objc-class-ref in libfltk.a(Fl_Quartz_Printer.mm.o)
"_OBJC_CLASS_$_NSPrintPanel", referenced from:
objc-class-ref in libfltk.a(Fl_Quartz_Printer.mm.o)
"_OBJC_CLASS_$_NSProcessInfo", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSScreen", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSString", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSTextView", referenced from:
_OBJC_CLASS_$_FLTextView in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSThread", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSView", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLView in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_CLASS_$_NSWindow", referenced from:
objc-class-ref in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLWindow in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_METACLASS_$_NSObject", referenced from:
_OBJC_METACLASS_$_FLWindow in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLTextView in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLWindowDelegate in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLWindowDelegateBefore10_6 in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLWindowDelegateBefore10_5 in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLAppDelegate in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLAppDelegateBefore10_5 in libfltk.a(Fl_cocoa.mm.o)
...
"_OBJC_METACLASS_$_NSTextView", referenced from:
_OBJC_METACLASS_$_FLTextView in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_METACLASS_$_NSView", referenced from:
_OBJC_METACLASS_$_FLView in libfltk.a(Fl_cocoa.mm.o)
"_OBJC_METACLASS_$_NSWindow", referenced from:
_OBJC_METACLASS_$_FLWindow in libfltk.a(Fl_cocoa.mm.o)
"_PMGetAdjustedPageRect", referenced from:
Fl_System_Printer::printable_rect(int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
Fl_System_Printer::start_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMGetFirstPage", referenced from:
Fl_System_Printer::start_job(int, int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMGetLastPage", referenced from:
Fl_System_Printer::start_job(int, int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMGetOrientation", referenced from:
Fl_System_Printer::margins(int*, int*, int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
Fl_System_Printer::start_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMGetPageFormatPaper", referenced from:
Fl_System_Printer::margins(int*, int*, int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
Fl_System_Printer::start_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMPaperGetMargins", referenced from:
Fl_System_Printer::margins(int*, int*, int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
Fl_System_Printer::start_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMSessionBeginCGDocumentNoDialog", referenced from:
Fl_System_Printer::start_job(int, int*, int*) in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMSessionBeginPageNoDialog", referenced from:
Fl_System_Printer::start_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMSessionEndDocumentNoDialog", referenced from:
Fl_System_Printer::end_job() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMSessionEndPageNoDialog", referenced from:
Fl_System_Printer::end_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMSessionError", referenced from:
Fl_System_Printer::end_job() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_PMSessionGetCGGraphicsContext", referenced from:
Fl_System_Printer::start_page() in libfltk.a(Fl_Quartz_Printer.mm.o)
"_TransformProcessType", referenced from:
foreground_and_activate() in libfltk.a(Fl_cocoa.mm.o)
"___CFConstantStringClassReference", referenced from:
CFString in libfltk.a(Fl_cocoa.mm.o)
CFString in libfltk.a(Fl_cocoa.mm.o)
CFString in libfltk.a(Fl_cocoa.mm.o)
CFString in libfltk.a(Fl_cocoa.mm.o)
CFString in libfltk.a(Fl_cocoa.mm.o)
CFString in libfltk.a(Fl_cocoa.mm.o)
CFString in libfltk.a(Fl_cocoa.mm.o)
...
"__objc_empty_cache", referenced from:
_OBJC_CLASS_$_FLWindow in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLWindow in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLTextView in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLTextView in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLWindowDelegate in libfltk.a(Fl_cocoa.mm.o)
_OBJC_METACLASS_$_FLWindowDelegateBefore10_6 in libfltk.a(Fl_cocoa.mm.o)
_OBJC_CLASS_$_FLWindowDelegate in libfltk.a(Fl_cocoa.mm.o)
...
"_kCFAllocatorDefault", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
Fl::copy(char const*, int, int, char const*) in libfltk.a(Fl_cocoa.mm.o)
Fl_X::dnd(int) in libfltk.a(Fl_cocoa.mm.o)
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
variation_selector_width(__CFString const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
...
"_kCFAllocatorNull", referenced from:
fl_mac_width(unsigned short const*, int, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
surrogate_width(unsigned short const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_kCFRunLoopDefaultMode", referenced from:
Fl::add_timeout(double, void (*)(void*), void*) in libfltk.a(Fl_cocoa.mm.o)
delete_timer(MacTimeout&) in libfltk.a(Fl_cocoa.mm.o)
"_kCFTypeDictionaryKeyCallBacks", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_kCFTypeDictionaryValueCallBacks", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_kCGColorSpaceGenericRGB", referenced from:
flcolortocgcolor(unsigned int) in libfltk.a(fl_font.cxx.o)
"_kCTFontAttributeName", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
Fl_Quartz_Graphics_Driver::text_extents(char const*, int, int&, int&, int&, int&) in libfltk.a(fl_font.cxx.o)
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
variation_selector_width(__CFString const*, Fl_Font_Descriptor*) in libfltk.a(fl_font.cxx.o)
"_kCTForegroundColorAttributeName", referenced from:
fl_mac_draw(char const*, int, float, float, Fl_Graphics_Driver*) in libfltk.a(fl_font.cxx.o)
"_kCTKernAttributeName", referenced from:
Fl_Font_Descriptor::Fl_Font_Descriptor(char const*, int) in libfltk.a(fl_font.cxx.o)
"_objc_alloc", referenced from:
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
+[FLTextView initialize] in libfltk.a(Fl_cocoa.mm.o)
+[FLWindowDelegate initialize] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindowDelegate fl_windowMiniaturize:] in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
-[FLAppDelegate application:openFile:] in libfltk.a(Fl_cocoa.mm.o)
createAppleMenu() in libfltk.a(Fl_cocoa.mm.o)
...
"_objc_alloc_init", referenced from:
Fl_X::calc_mac_os_version() in libfltk.a(Fl_cocoa.mm.o)
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
DataReady::DataReadyThread(void*) in libfltk.a(Fl_cocoa.mm.o)
fl_mac_flush_and_wait(double) in libfltk.a(Fl_cocoa.mm.o)
createAppleMenu() in libfltk.a(Fl_cocoa.mm.o)
get_window_frame_sizes(int&, int&, int&) in libfltk.a(Fl_cocoa.mm.o)
+[FLTextInputContext initialize] in libfltk.a(Fl_cocoa.mm.o)
...
"_objc_autorelease", referenced from:
-[FLWindowDelegate fl_windowMiniaturize:] in libfltk.a(Fl_cocoa.mm.o)
-[FLaboutItemTarget showPanel] in libfltk.a(Fl_cocoa.mm.o)
CGBitmapContextToNSImage(CGContext*) in libfltk.a(Fl_cocoa.mm.o)
"_objc_msgSend", referenced from:
Fl_X::calc_mac_os_version() in libfltk.a(Fl_cocoa.mm.o)
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
DataReady::DataReadyThread(void*) in libfltk.a(Fl_cocoa.mm.o)
fl_ready() in libfltk.a(Fl_cocoa.mm.o)
-[FLWindow convertBaseToScreen:] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindow initWithFl_W:contentRect:styleMask:] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindow recursivelySendToSubwindows:] in libfltk.a(Fl_cocoa.mm.o)
...
"_objc_msgSendSuper2", referenced from:
-[FLWindow initWithFl_W:contentRect:styleMask:] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindowDelegateBefore10_5 windowDidDeminiaturize:] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindowDelegateBefore10_5 windowWillMiniaturize:] in libfltk.a(Fl_cocoa.mm.o)
-[FLGLViewLayer did_view_resolution_change] in libfltk.a(Fl_cocoa.mm.o)
-[FLViewLayer did_view_resolution_change] in libfltk.a(Fl_cocoa.mm.o)
-[FLViewLayer dealloc] in libfltk.a(Fl_cocoa.mm.o)
-[FLView initWithFrame:] in libfltk.a(Fl_cocoa.mm.o)
...
"_objc_msgSendSuper2_stret", referenced from:
-[FLWindow constrainFrameRect:toScreen:] in libfltk.a(Fl_cocoa.mm.o)
"_objc_msgSend_stret", referenced from:
Fl_X::calc_mac_os_version() in libfltk.a(Fl_cocoa.mm.o)
fl_open_display() in libfltk.a(Fl_cocoa.mm.o)
-[FLWindow convertBaseToScreen:] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindow setSubwindowFrame] in libfltk.a(Fl_cocoa.mm.o)
-[FLWindowDelegate windowDidMove:] in libfltk.a(Fl_cocoa.mm.o)
update_e_xy_and_e_xy_root(NSWindow*) in libfltk.a(Fl_cocoa.mm.o)
-[FLWindowDelegate windowDidResize:] in libfltk.a(Fl_cocoa.mm.o)
...
"_objc_opt_class", referenced from:
-[FLWindow recursivelySendToSubwindows:] in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
-[FLAppDelegate applicationDidChangeScreenParameters:] in libfltk.a(Fl_cocoa.mm.o)
+[FLApplication sendEvent:] in libfltk.a(Fl_cocoa.mm.o)
-[FLView insertText:replacementRange:] in libfltk.a(Fl_cocoa.mm.o)
-[FLView setMarkedText:selectedRange:replacementRange:] in libfltk.a(Fl_cocoa.mm.o)
"_objc_opt_isKindOfClass", referenced from:
-[FLWindow recursivelySendToSubwindows:] in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
-[FLAppDelegate applicationDidChangeScreenParameters:] in libfltk.a(Fl_cocoa.mm.o)
+[FLApplication sendEvent:] in libfltk.a(Fl_cocoa.mm.o)
-[FLView insertText:replacementRange:] in libfltk.a(Fl_cocoa.mm.o)
-[FLView setMarkedText:selectedRange:replacementRange:] in libfltk.a(Fl_cocoa.mm.o)
"_objc_opt_self", referenced from:
+[FLWindowDelegate initialize] in libfltk.a(Fl_cocoa.mm.o)
"_objc_release", referenced from:
Fl_X::calc_mac_os_version() in libfltk.a(Fl_cocoa.mm.o)
DataReady::DataReadyThread(void*) in libfltk.a(Fl_cocoa.mm.o)
fl_mac_flush_and_wait(double) in libfltk.a(Fl_cocoa.mm.o)
drain_dropped_files_list() in libfltk.a(Fl_cocoa.mm.o)
-[FLWindowDelegate fl_windowMiniaturize:] in libfltk.a(Fl_cocoa.mm.o)
rect_to_NSBitmapImageRep(Fl_Window*, int, int, int, int) in libfltk.a(Fl_cocoa.mm.o)
createAppleMenu() in libfltk.a(Fl_cocoa.mm.o)
...
"_objc_retain", referenced from:
Fl_X::set_cursor(Fl_Cursor) in libfltk.a(Fl_cocoa.mm.o)
Fl_Paged_Device::print_window(Fl_Window*, int, int) in libfltk.a(Fl_cocoa.mm.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [pp] Error 1
make[2]: *** [CMakeFiles/pp.dir/all] Error 2
make[1]: *** [CMakeFiles/pp.dir/rule] Error 2
make: *** [pp] Error 2
```

**Proposed solution**
Would be nice if the example would build

**Describe alternatives you've considered**
I'm trying other cross-platform GUI frameworks that vcpkg 'supports', e.g., see https://github.com/microsoft/vcpkg/issues/17052

--- a/ports/fltk/CONTROL
+++ b/ports/fltk/CONTROL
@@ -1,6 +1,0 @@
-Source: fltk
-Version: 1.3.5
-Port-Version: 3
-Homepage: https://www.fltk.org/
-Description: FLTK (pronounced fulltick) is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows, and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL and its built-in GLUT emulation.
-Build-Depends: zlib, libpng, libjpeg-turbo

--- a/ports/fltk/fix-system-link.patch
+++ b/ports/fltk/fix-system-link.patch
@@ -1,0 +1,18 @@
+diff --git a/CMake/macros.cmake b/CMake/macros.cmake
+index 0c2586f..eb77510 100644
+--- a/CMake/macros.cmake
++++ b/CMake/macros.cmake
+@@ -37,6 +37,13 @@ macro(FL_ADD_LIBRARY LIBNAME LIBTYPE LIBFILES)
+     add_library(${LIBRARY_NAME} ${LIBTYPE} ${LIBFILES})
+ 
+     target_include_directories(${LIBRARY_NAME} PUBLIC $<INSTALL_INTERFACE:include>)
++    
++    if (APPLE)
++        find_library(Cocoa Cocoa)
++        target_link_libraries(${LIBRARY_NAME} PUBLIC $<$<PLATFORM_ID:Darwin>:${Cocoa}>)
++    elseif (WIN32)
++        target_link_libraries(${LIBRARY_NAME} PUBLIC gdi32 gdiplus user32 advapi32 ole32 shell32 comdlg32)
++    endif()
+ 
+     set_target_properties(${LIBRARY_NAME}
+         PROPERTIES

--- a/ports/fltk/portfile.cmake
+++ b/ports/fltk/portfile.cmake
@@ -1,11 +1,11 @@
+# FLTK has many improperly shared global variables that get duplicated into every DLL
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
 vcpkg_download_distfile(ARCHIVE
     URLS "https://fltk.org/pub/fltk/1.3.5/fltk-1.3.5-source.tar.gz"
     FILENAME "fltk-1.3.5.tar.gz"
     SHA512 db7ea7c5f3489195a48216037b9371a50f1119ae7692d66f71b6711e5ccf78814670581bae015e408dee15c4bba921728309372c1cffc90113cdc092e8540821
 )
-
-# FLTK has many improperly shared global variables that get duplicated into every DLL
-vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
@@ -15,6 +15,7 @@ vcpkg_extract_source_archive_ex(
         add-link-libraries.patch
         config-path.patch
         include.patch
+        fix-system-link.patch
 )
 
 if (VCPKG_TARGET_ARCHITECTURE MATCHES "arm" OR VCPKG_TARGET_ARCHITECTURE MATCHES "arm64")
@@ -42,6 +43,8 @@ vcpkg_install_cmake()
 
 vcpkg_fixup_cmake_targets(CONFIG_PATH share/fltk)
 
+vcpkg_copy_pdbs()
+
 if(VCPKG_TARGET_IS_OSX)
     vcpkg_copy_tools(TOOL_NAMES fluid.app fltk-config AUTO_CLEAN)
 elseif(VCPKG_TARGET_IS_WINDOWS)
@@ -50,8 +53,6 @@ elseif(VCPKG_TARGET_IS_WINDOWS)
 else()
     vcpkg_copy_tools(TOOL_NAMES fluid fltk-config AUTO_CLEAN)
 endif()
-
-vcpkg_copy_pdbs()
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
     file(REMOVE_RECURSE
@@ -74,4 +75,4 @@ foreach(FILE Fl_Export.H fl_utf8.h)
     file(WRITE ${CURRENT_PACKAGES_DIR}/include/FL/${FILE} "${FLTK_HEADER}")
 endforeach()
 
-file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/fltk/vcpkg.json
+++ b/ports/fltk/vcpkg.json
@@ -1,0 +1,12 @@
+{
+  "name": "fltk",
+  "version": "1.3.5",
+  "port-version": 4,
+  "description": "FLTK (pronounced fulltick) is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows, and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL and its built-in GLUT emulation.",
+  "homepage": "https://www.fltk.org/",
+  "dependencies": [
+    "libjpeg-turbo",
+    "libpng",
+    "zlib"
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1982,7 +1982,7 @@
     },
     "fltk": {
       "baseline": "1.3.5",
-      "port-version": 3
+      "port-version": 4
     },
     "fluidlite": {
       "baseline": "2020-08-27",

--- a/versions/f-/fltk.json
+++ b/versions/f-/fltk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "09b2ceac638353fa15310751f007a9e868087460",
+      "version": "1.3.5",
+      "port-version": 4
+    },
+    {
       "git-tree": "4f56d4551d634591725cc016e804323a7a793faa",
       "version-string": "1.3.5",
       "port-version": 3

